editor.planx.uk/src/@planx/components/Send/Editor.tsx

```diff
@@ -45,26 +48,45 @@ const SendComponent: React.FC<Props> = (props) => {
     }),
   });
 
-  const options: { value: SendIntegration; label: string }[] = [
+  const options: {
+    value: SendIntegration;
+    label: string;
+    description?: string;
+    disabled?: boolean;
+  }[] = [
     {
       value: "bops",
-      label: "BOPS",
-    },
-    {
-      value: "uniform",
-      label: "Uniform",
-    },
-    {
-      value: "idox",
-      label: "Idox Nexus (TESTING ONLY)",
+      label: `BOPS ${
+        import.meta.env.VITE_APP_ENV === "production" ? "production" : "staging"
+      }`,
+      // TODO add BOPS to state.teamIntegrations type and disable accordingly
     },
     {
       value: "email",
-      label: "Email to planning office",
+      label: `Email to ${useStore(
+        (state) => state.teamSettings.submissionEmail || "planning office",
+      )}`,
+      description:
+        "Configure one submission email per account under Team Settings. Please setup redirect rules in your inbox if different services require different emails.",
     },
     {
       value: "s3",
-      label: "Upload to AWS S3 bucket",
+      label: "Microsoft Sharepoint",
+      description:
+        "Receive submissions direct in your internal files system using a Power Automate workflow",
+      // TODO add Webhook URL to state.teamIntegrations type and disable accordingly
+    },
+    {
+      value: "uniform",
+      label: `Uniform ${
+        import.meta.env.VITE_APP_ENV === "production" ? "production" : "staging"
+      }`,
+      description:
+        "Warning: Legacy integration with very limited support! Only suitable for use with LDC-E and LDC-P application types for select councils.",
+      disabled: useStore(
+        (state) =>
+          !["buckinghamshire", "lambeth", "southwark"].includes(state.teamSlug),
+      ),
```

Comment:
There's a few levels of validation possible here that I'm still ironing out: 
- Formik validation in this modal - currently only responsible for ensuring at least one option is checked
- Disabling an individual checkbox - first done globally based on permissions, then done per "rules" specified per option here
- Validation checks on publish - another option for checking if your selections match your actual configured intergrations ?